packages/backup-restore-operator/charts/templates/_helpers.tpl

```diff
@@ -57,4 +57,31 @@ Create the name of the service account to use
 
 {{- define "backupRestore.pvcName" -}}
 {{ include "backupRestore.fullname" . }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+{{- define "windows-node-selector" -}}
+{{- if semverCompare "<1.14-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
I know this was in example, but by 2.5.0 we will be shipping 1.16+, so that greatly simplifies it. We could break out of helper, up to you.